CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Contributing to ngrok-go
+
+Thank you for deciding to contribute to ngrok-go. The easiest way to contribute is to [open a new issue](https://github.com/ngrok/ngrok-go/issues/new) with clear repro steps when you find an issue with the library. We will triage and investigate these issues at a regular interval.
```

Comment:
It may be worth splitting out:

```
## Reporting a bug

When reporting a bug, please include clear reproduction steps.

## Contributing code

Bugfixes and small improvements are always appreciated!

For any larger changes or features, please open an issue first to discuss whether the change makes sense. When in doubt, it's always okay to open an issue first.
```

It probably needs more word-smithing, but I think there's value in separating out reporting a bug from contributing code

Reply:
updated!